packages/elements/src/photon-dropdown/photon-dropdown.tsx

```diff
@@ -332,11 +332,11 @@ export const PhotonDropdown = <T extends { id: string }>(props: {
             class="bg-white overflow-x-hidden overflow-y-auto relative"
             style={{
               'max-height': '200px',
-              'min-height': '56px',
+              'min-height': '37px',
```

Comment:
TBH, I have no clue why this was previously 56 or why using the virtualization thing change caused this... 🤷